packages/serverpod_test/lib/src/with_serverpod.dart

```diff
@@ -49,6 +63,50 @@ typedef TestClosure<T> = void Function(
   TestSession testSession,
 );
 
+/// Internal helper extension for the [DatabaseTestConfig] class.
+extension ConfigRuntimeChecks on DatabaseTestConfig {
+  /// Returns true if rollbacks are disabled.
+  bool get areRollbacksDisabled => _rollbackDatabase == _RollbackDatabase.never;
+}
```

Comment:
Question: What is the motivation behind this indirection?

Reply:
It was needed to get the current setting, but will revert these changes: https://github.com/serverpod/serverpod/pull/2791#discussion_r1782467043